lib/Doctrine/MongoDB/EagerCursor.php

```diff
@@ -107,7 +121,7 @@ public function getSingleResult()
     public function initialize()
     {
         if ($this->initialized === false) {
-            $this->data = $this->cursor->toArray();
+            $this->data = $this->cursor->toArray($this->useKeys);
```

Comment:
Going to change this to just have a `$useKeys` param on `initialize()`. We don't need it on the constructor.